.travis.yml

```
@@ -1,61 +1,45 @@
 language: python
 python: 3.6.6
 
-env:
-  global:
-    - TRAVIS_NODE_VERSION=8.9.4
-    - PGPORT=5433
-
-dist: trusty
-sudo: true
+sudo: required
 
 addons:
-  hosts:
-    - warehouse.local
-  postgresql: 10
   apt:
     packages:
-    - postgresql-10
-    - postgresql-client-10
-    - nasm
+      - docker-ce
 
 services:
-  - postgresql
+  - docker
 
 cache:
```

Travis CI is more aware of what they suggest to cache for pip:
```suggestion
 cache:
   pip: true
```